### PR TITLE
NYS-66: adds a new manually triggered github action to refresh the tr…

### PR DESCRIPTION
**Pull request by joelsteidl**
This is a simple PR with just adding a new action file. I'll check in with the NYS team to see when we can test.

### CODE REVIEW

File: .github/workflows/pantheon-refresh-training.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+      - name: Disable Pantheon SSH strict host key checking
+        run: 'printf "\nHost *\n\tStrictHostKeyChecking no" >> ~/.ssh/config'
+
+      - name: Install Terminus
+        uses: pantheon-systems/terminus-github-actions@main
+        with:
+          pantheon-machine-token: ${{ secrets.PANTHEON_MACHINE_TOKEN }}
+
+      - name: Merge the latest code from dev (master) into training.
```
Comment (routinet) [inline .github/workflows/pantheon-refresh-training.yml:34]: Is there a reason we would not merge direct from `live` instead of `dev`?

**Comment by routinet** [comment]
@joelsteidl approved, but is it possible to make this more robust by allowing for selection/specification of the target multidev?  This is primarily intended to keep `training` up-to-date, but I could see where this would be useful for other environments also.

**Comment by joelsteidl** [comment]
@routinet That would be relatively easy but the name of the action should probably be "Pantheon Refresh Multidev" and then set `training` as the default. We'd also need to block the ability to refresh the live environment (not even sure that is possible).

**Comment by joelsteidl** [comment]
@routinet I tested this on a sandbox project but ended up making this a lot more flexible.

* User can choose a Sync Type (Code + Database or Database Only)

**Comment by joelsteidl** [comment]
@routinet One more...here is what the Github UI looks like. https://screenshots.atendesigngroup.com/3VzntRMF

**Pull request merged**